docs/UserGuide.md

```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
 
 ### 3.2 Application Home Window Breakdown
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
```

Comment:
The alt text (in between `[]`) should be normal text, like Main Window Screenshot. No need for PascalCase :)